main.py

```diff
@@ -121,6 +189,12 @@ def _get_artist_name_from_submission_title(post_title):
             return artist.group('artist')
 
         raise LookupError
+    
+    #NOTE, this function is a placeholder, and currently not functioning!
+    @staticmethod
+    def _get_song_name_from_submission_title(post_title):
+        return post_title
```

Comment:
Need to write something here (maybe regex?) to help parse the song name.

Should add some tests similar to the artist matching tests to ensure we are parsing it properly.

Reply:
Added, please check it out and make a new comment if you find issues with it. Resolving.